Doc/library/socket.rst

```diff
@@ -482,6 +482,11 @@ The following functions all create :ref:`socket objects <socket-objects>`.
    .. versionchanged:: 3.7
        The CAN_ISOTP protocol was added.
 
+   .. versionchanged:: 3.7
+      On Linux: when :const:`SOCK_NONBLOCK` or :const:`SOCK_CLOEXEC` bit
+      flags are applied to *type* they are cleared, and :attr:`socket.type`
+      will not reflect them.
```

Comment:
This belongs with the _socket.type_ attribute, not the constructors. It sounds like Python will prevent the caller from passing these flags to Linux, and makes you wonder if _socketpair_ and _fromfd_ are affected.

Reply:
Actually they will be passed to Linux `socket()`, but won't be applied to Python `sock.type` attribute.  I'll rephrase this.